README.md

```diff
@@ -38,3 +38,16 @@ When to Modify Tables:
 5. If there are still problems. Find the source of the problematic file and replace all tokens that IWYU associates with that file with a better file. An example of this is asm-generic/percpu.h. It has many tokens that need to be manually associated with linux/percpu.h. Add the following line to symbol.imp force an association between a token and a header:
    ```{ symbol: ["TOKEN", private, "asm/SOLUTION-FILE", public]},```
    Examples are found in symbol.imp
+
+## auto_iwyu.py
+```
+python auto-iwyu.py -c compile_commands.json
+```
+The auto-iwyu script will attempt to build your whole compile_commands.json file and make changes automatically.
+
+
+## build_intermediary.py
+```
+python build_intermediary.py -c compile_commands.json -j $(nproc)
+```
+The build intermediary file will build files with a specific file extension for the entire compile_commands.json. The default extension is .i but can be changed with the -e flag. Pass in whatever arguments are necessary with the -a flag the default is -E.
```

Comment:
Can you provide an example in the readme of `-e`? Is it `-e s` or `-e .s` or something else, for example?

---

`-a` seems odd, perhaps you can infer it from the file extension?

`.i` -> `-E`
`.s` -> `-S`
`.ll` -> `-emit-llvm -S`
`.bc` -> `-emit-llvm`

Not sure what else we'd need. Perhaps `-a` is too flexible?

---

> Pass in whatever arguments are necessary with the -a flag the default is -E.

Needs punctuation after `flag`.